README.md

```diff
@@ -2,56 +2,10 @@
 
 ## Background
 
-We will be building an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture.
+As `.env.*` is added to the `.gitignore`, anyone who wishes to clone this repo will not have access to the necessary environment variables. 
 
-Your database will be PSQL, and you will interact with it using [node-postgres](https://node-postgres.com/).
+You will need to create two .env files for your project: `.env.test` and `.env.development`. Into each, add `PGDATABASE=<database_name_here>`, with the correct database name for that environment (see `/db/setup.sql` for the database names). If you need any help, please reference documentation
```

Comment:
This is the documentation for your app, so should be the help they refer to. 

Instead of getting them to search around for names, i would just tell them what they need to call the PGDATABASE names. 
It's more likely to go wrong if you're vague. 

When writing readmes, think about whether you could follow this correctly at the start of the week, and make it as easy as possible.